libs/safe/src/lib/components/widgets/summary-card-settings/summary-card-general/summary-card-general.component.ts

```diff
@@ -36,7 +36,7 @@ import {
 import { Dialog } from '@angular/cdk/dialog';
 
 /** Default number of resources to be fetched per page */
-const ITEMS_PER_PAGE = 10;
+const ITEMS_PER_PAGE = 30;
```

Comment:
This should be 10, no?
```suggestion
const ITEMS_PER_PAGE = 10;
```